server.js

```diff
@@ -19,10 +17,10 @@ app.use(
   express.static(path.join(__dirname, "client/storybook-static"))
 );
 
-app.get("/api/locations/:tour", async (req, res) => {
-  const { tour } = req.params;
+app.get("/api/locations/name/:name", async (req, res) => {
+  const { name } = req.params;
```

Comment:
If you want to search through your locations for a specific name, you could use queries instead.
```suggestion
app.get("/api/locations", async (req, res) => {
  const { name } = req.query;
```

In that case you would need to change your route in your api/locations.js.

Reply:
Implemented 🙌 THX for suggesting this.